INSTALL.rst

```diff
@@ -87,65 +89,80 @@ For example, on Debian GNU/Linux, you can install them as follows:
                         rabbitmq-server \
                         redis-server
 
-and do a system install for the Sass preprocessor by following `Sass web guide <http://sass-lang.com/install>`_ and running:
+Now, add the following lines in your "elasticsearch.yml" (for
+Debian GNU/Linux the full path is
+``/etc/elasticsearch/elasticsearch.yml``):
 
 .. code-block:: shell
 
-  sudo npm install -g sass node-sass clean-css uglify-js requirejs
-
+  # MY CONFIGS
 
-Installation
-^^^^^^^^^^^^
+  cluster.name: cap
+  discovery.zen.ping.multicast.enabled: false
+  http.port: 9200
+  http.publish_port: 9200
 
-Let's start by creating a new virtual environment that will hold our CAP
-v3.0 instance:
+Finally, do a system install (see below for how to do a local install
+instead) for the Sass preprocessor by following
+`Sass web guide <http://sass-lang.com/install>`_ and running:
 
 .. code-block:: shell
 
-   mkvirtualenv cap
+  sudo npm install -g node-sass clean-css uglify-js requirejs
 
-Start redis server in the background:
+Installation
+^^^^^^^^^^^^
+
+Let's start by cloning the repository:
```

Comment:
There's a lot more power in virtualenv if we don't do that. It will allow us to switch virtualenvs easily without having to clone again each time and we avoid a deep directory structure. I've been using it like that for a while and it makes things a lot easier. There's no downside to it as far as I can tell.